mathesar/users/decorators.py

```diff
@@ -0,0 +1,30 @@
+import functools
+
+from django.contrib.auth import get_user_model
+from django.shortcuts import redirect
+from django.urls import reverse
+
+
+def superuser_must_not_exists(view_function=None):
```

Comment:
Okay, thank you for the explanation. My only request now is to make the following grammar change:

```suggestion
def superuser_must_not_exist(view_function=None):
```

(and also at the call)